### PR TITLE
update webrings

### PR DESCRIPTION
**Pull request by tanishq-soni**
Added myself to webrings..! :)

### CODE REVIEW

File: public/members.json [hunk 1]
```diff
@@ -42,5 +42,9 @@
   {
     "member": "Pragya Rajpurohit",
     "url": "https://pragyaraj.tech/index1.html"
+  },
+  {
+    "member": "Tanishq Soni",
+    "url": "https://tanishqsoni.me"
```
Comment (sampoder) [inline public/members.json:48]: This site doesn't seem to have the widget

Comment (tanishq-soni) [reply public/members.json:48]: which widget are u talking about?

Comment (sampoder) [reply public/members.json:48]: read the instructions here: https://webring.hackclub.com

Comment (tanishq-soni) [reply public/members.json:48]: can you check it now, at the bottom of the website !

Comment (sampoder) [reply public/members.json:48]: should be good

**Review by sampoder — APPROVED** [review]

**Comment by googol88** [comment]
Looks good, I'll be merging the PR now. Thanks @sampoder for reviewing!

**Pull request merged**